launcher/ui/pages/modplatform/atlauncher/AtlOptionalModDialog.cpp

```diff
@@ -212,6 +301,8 @@ AtlOptionalModDialog::AtlOptionalModDialog(QWidget *parent, QVector<ATLauncher::
     ui->treeView->header()->setSectionResizeMode(
             AtlOptionalModListModel::DescriptionColumn, QHeaderView::Stretch);
 
+    connect(ui->shareCodeButton, &QPushButton::pressed,
+        this, &AtlOptionalModDialog::useShareCode);
     connect(ui->selectRecommendedButton, &QPushButton::pressed,
             listModel, &AtlOptionalModListModel::selectRecommended);
     connect(ui->clearAllButton, &QPushButton::pressed,
```

Comment:
Not really your fault I think, but why aren't these all `QPushButton::clicked`?

Reply:
In this case, I would have gone with what was already in use. Though I believe I would have written the original code (based on the filename), and I'm not sure - should probably be clicked, as I recall from documentation.

I'll take a look later when at a workstation.

Reply:
Been a long time since I've used the ATLauncher - but I found 6ANo8pGL as an example (Resonant Rise 3.4.0.3) on their GitHub.